seed/views/v3/import_files.py

```diff
@@ -627,23 +627,21 @@ def start_save_data(self, request, pk=None):
                     pk)}, status=status.HTTP_400_BAD_REQUEST)
 
         cycle_id = body.get('cycle_id')
+        multiple_cycle_upload = body.get('multiple_cycle_upload', False)
```

Comment:
I would think `cycle_id == "year_ending"` would be enough to commucate we are doing multicycle, is there a reason we  need `multiple_cycle_upload` too?

Reply:
Now that we are letting the users set a default cycle I'm using cycle_id to save the default cycle. Therefore, I decided to add a new boolean field to import_file to save the fact that we are doing multicycle. I'm open to suggestions though.